js/iosOverlay.js

```diff
@@ -102,8 +104,18 @@ var iosOverlay = function(params) {
 			});
 		}
 
+		if(settings.mouseover) {
+			$(overlayDOM).mouseover( function () {
+			  clearTimeout(hideTimeOut);
```

Comment:
Not a blocker, but a consistency point:

You've used `window.setTimout()`, but then used `clearTimeout()` without a `window` prefix. Not that it particularly matters either way.